qtodotxt2/main_controller.py

```diff
@@ -325,3 +326,10 @@ def completeTasks(self, tasks):
             else:
                 task.setPending()
 
+    docPathChanged = QtCore.pyqtSignal()
+
+    @QtCore.pyqtProperty('QUrl', notify=docPathChanged)
```

Comment:
I am not sure what docpath is, but it is typically something that doe not change so set notify to None or whatever else to disable it, instead of creating a new signal